include/picongpu/plugins/IsaacPlugin.hpp

```diff
@@ -27,6 +27,7 @@
 #include <pmacc/dataManagement/DataConnector.hpp>
 #include <pmacc/static_assert.hpp>
 
+#define ISAAC_IDX_TYPE cupla::IdxType
```

Comment:
Is this a workaround or the common way to set the index type for ISAAC?
Is it possible to extract the index type from one of the classes passed to `IsaacVisualization<>`?

Reply:
This is the new common way, as ISAAC did not have this feature before. The problem is, that some internal types are using the index type, too. So without this solution (yes, macros are ugly) I would break backwards compatibility. You can't have two different visualization objects with different index types atm but before you were not able to change the index type at all.